applications/serial_lte_modem/doc/CMUX_AT_commands.rst

```diff
@@ -0,0 +1,115 @@
+.. _SLM_AT_CMUX:
+
+CMUX AT commands
+****************
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page describes CMUX-related AT commands.
```

Comment:
This could have a short explanation of cmux and why it was added. I think that many readers will come through here and miss the other places that it was shortly introduced.

Reply:
Agreed with this. A sentence or two would probably be enough.